examples/notebooks/TextCNN.ipynb

```diff
@@ -1,853 +1,1016 @@
+        "\n",
+        "Lets get started! "
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "sovYyC0Zznj-"
+      },
+      "source": [
+        "## Required Dependencies \n",
+        "\n",
+        "In this example we only need torchtext and spacy package, assuming that `torch` and `ignite` are already installed. We can install it using `pip`:\n",
+        "\n",
+        "`pip install torchtext==0.9.1 spacy`\n",
+        "\n",
+        "`python -m spacy download en_core_web_sm`"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "7XHAD9x7znj_"
+      },
+      "source": [
+        "!pip install pytorch-ignite torchtext==0.9.1 spacy\n",
+        "!python -m spacy download en_core_web_sm"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "lZty7-RWznkA"
+      },
+      "source": [
+        "## Import Libraries"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "VKTazeAkznkB"
+      },
+      "source": [
+        "import random"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "wxThg0YTznkD"
+      },
+      "source": [
+        "`torchtext` is a library that provides multiple datasets for NLP tasks, similar to `torchvision`. Below we import the following:\n",
+        "* **data**: A module to setup the data in the form Fields and Labels.\n",
+        "* **datasets**: A module to download NLP datasets.\n",
+        "* **GloVe**: A module to download and use pretrained GloVe embedings."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "XrXE-f7jznkD"
+      },
+      "source": [
+        "from torchtext import data\n",
+        "from torchtext import datasets\n",
+        "from torchtext.vocab import GloVe"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ivAnTyEfznkE"
+      },
+      "source": [
+        "We import torch, nn and functional modules to create our models! "
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "gbEFAWr0znkE"
+      },
+      "source": [
+        "import torch\n",
+        "import torch.nn as nn\n",
+        "import torch.nn.functional as F\n",
+        "\n",
+        "SEED = 1234\n",
+        "random.seed(SEED)\n",
+        "torch.manual_seed(SEED)\n",
+        "torch.cuda.manual_seed(SEED)"
```

Comment:
Let's update here as well. We can use `ignite.utils.manual_seed(SEED)` instead of these 3 lines.